### PR TITLE
disabling squeeze/unsqueeze; disabling BN/BN_BWD for perf concern

### LINKED ISSUE #76883: NVFuser internal failures
### 🐛 Describe the bug

- [ ] graph 0 (hits fallback)
- [ ] graph 1 (segfault)

Repro script:

```python
import argparse
import torch
import torch.utils.jit.log_extract as log_extract

ir = ["""graph(%pad_sequence_embeddings_7.1 : Half(10, 260, 192, strides=[49920, 192, 1], requires_grad=0, device=cuda:0),
      %pad_sequence_embeddings_3.1 : Half(10, 1, 192, strides=[192, 192, 1], requires_grad=0, device=cuda:0)):
  %2 : int = prim::Constant[value=1]()
  %6 : Half(10, 192, strides=[192, 1], requires_grad=0, device=cuda:0) = aten::squeeze(%pad_sequence_embeddings_3.1, %2)
  %7 : Half(10, 1, 192, strides=[192, 192, 1], requires_grad=0, device=cuda:0) = aten::unsqueeze(%6, %2)
  %mul_1.1 : Half(10, 260, 192, strides=[49920, 192, 1], requires_grad=0, device=cuda:0) = aten::mul(%pad_sequence_embeddings_7.1, %7)
  return (%mul_1.1, %7)
""", """graph(%weight.132 : Half(1024, strides=[1], requires_grad=0, device=cuda:0),
      %bias.78 : Half(1024, strides=[1], requires_grad=0, device=cuda:0),
      %bmm_9.1 : Half(10, 1024, 1, strides=[1024, 1, 1], requires_grad=0, device=cuda:0)):
  %3 : bool = prim::Constant[value=1]()
  %4 : float = prim::Constant[value=1.0000000000000001e-05]()
  %5 : int[] = prim::Constant[value=[1024]]()
  %6 : int = prim::Constant[value=-1]()
  %11 : Half(10, 1024, strides=[1024, 1], requires_grad=0, device=cuda:0) = aten::squeeze(%bmm_9.1, %6)
  %main_module_over_arch_1_norm_0.1 : Half(10, 1024, strides=[1024, 1], requires_grad=0, device=cuda:0) = aten::layer_norm(%11, %5, %weight.132, %bias.78, %4, %3)
  %main_module_over_arch_1_norm_1.1 : Half(10, 1024, strides=[1024, 1], requires_grad=0, device=cuda:0) = aten::sigmoid(%main_module_over_arch_1_norm_0.1)
  %mul_21.1 : Half(10, 1024, strides=[1024, 1], requires_grad=0, device=cuda:0) = aten::mul(%11, %main_module_over_arch_1_norm_1.1)
  return (%mul_21.1)
"""]

parser = argparse.ArgumentParser(description='Type the index of the ir graph to run, or -1 to run all')
parser.add_argument('id', type=int)
args = parser.parse_args()

def run_on(ir):
    _, inputs = log_extract.load_graph_and_inputs(ir)
    log_extract.run_nvfuser(ir, inputs)

if args.id == -1:
    for i, s in enumerate(ir):
        print('graph', i)
        run_on(s)
else:
    run_on(ir[args.id])
```

### Versions

A100, master

### PR DESCRIPTION
**Pull request by jjsjann123**
Fixes #76883 (via disabling squeeze/unsqueeze)

Disabling BN fwd/bwd for our perf concern. I need to update our python tests. Awaiting build to finish so I can update tests accordingly.

### CODE REVIEW

**Comment by jjsjann123** [comment]
Tests passed locally. CC'ing @csarofeen on the BN stuff (in case we needed last minute revert).

**Review by csarofeen — APPROVED** [review]
Thanks.

**Comment by jjsjann123** [comment]
The failure `test_nvfuser_correctness_add_cuda_bool` seems to have been marked as `expected failure` in my latest upstream container. I'll try merge upstream, meanwhile I think this PR is good for review/merge.

File: test/test_jit_cuda_fuser.py [hunk 1]
```diff
@@ -1631,30 +1631,50 @@ def t_wb(shapes: List[int], x, w, b, eps: float, cudnn: bool):
     @unittest.skipIf(GRAPH_EXECUTOR != ProfilingMode.PROFILING,
                      "Requires fusion optimization pass to be effective")
     def test_norm_half_layer(self):
+        op = "aten::batch_norm"
```
Comment (davidberard98) [inline test/test_jit_cuda_fuser.py:1634]: curious if it makes sense to just put these un-skips in setUp()?

**Review by davidberard98 — APPROVED** [review]
LGTM!

File: test/test_jit_cuda_fuser.py (hunk 1 above)
Comment (jjsjann123) [reply test/test_jit_cuda_fuser.py:1634]: That's actually a good idea. Would be easier to revert later. Will do

**Comment by jjsjann123** [comment]
Tests passed locally. I'll merge once CI finishes :crossed_fingers:

**Comment by jjsjann123** [comment]
Nice, we actually got a all green CI :tada:

**Pull request closed without merging**